Runtimes/Overlay/Cxx/CMakeLists.txt

```diff
@@ -0,0 +1,45 @@
+
+if(NOT APPLE)
+  add_subdirectory(cxxshim)
+endif()
+if(LINUX)
```

Comment:
I actually need to target a Linux that does not have a libstdc++, but uses libcxx instead. This assumption is incorrect.